templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>Mirroring my comment in the pixel clustering notebook, to match the output of Segment_Image_Data.ipynb, I believe <code>cell_table_name</code> would be something like <code>os.path.join("single_cell_output","cell_table_size_normalized.csv")</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Mirroring Noah's comment, we'll be doing a complete retool of the example data repo so that one single dataset is used throughout all <code>ark</code>  notebooks; this will be a part of that overhaul.</p>